<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeetingRoomDialog</class>
 <widget class="QDialog" name="MeetingRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1869</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>910</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>statusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="clientCountLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>clientCountLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startMeetingButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>startMeetingButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMeetingButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>closeMeetingButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>731</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="clientListWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="broadcastButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>820</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kickButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>踢出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startVideoButton">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>60</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopVideoButton">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>60</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="localVideoLabel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>350</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="remoteVideoLabel">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>340</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendImageButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发送图片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
